packages/plugins/graphql/server/services/builders/input.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 const { inputObjectType, nonNull } = require('nexus');
+const {
+  contentTypes: { getWritableAttributes },
```

Comment:
there is an isWritableAttribute in the utils already. you might want to use it

Reply:
Yup, done